node/node-pakiti2-json.py

```diff
@@ -37,9 +38,9 @@
     print(response.read())
 
   except:
-    print('[]')
+    print('Cannot Connect')
```

Comment:
Except it's PHP that calls this script...
```php
    $result  = exec("/usr/bin/python node/node-pakiti2-json.py $NODE");
    $result  = str_replace("'", '"', $result);
    $result = json_decode($result, True);
```
What happens when you call `json_decode('Cannot Connect')`?